general/general_instance.json

```diff
@@ -0,0 +1,99 @@
+{
+  "@context": {
+    "xsd": "http://www.w3.org/2001/XMLSchema#",
+    "general": "https://github.com/GAIA-X4PLC-AAD/ontology-management-base/tree/main/general/",
+    "gx": "https://registry.lab.gaia-x.eu/development/api/trusted-shape-registry/v1/shapes/jsonld/trustframework#",
+    "skos": "http://www.w3.org/2004/02/skos/core#",
+    "sh": "http://www.w3.org/ns/shacl#"
+  },
+  "@id": "did:web:registry.gaia-x.eu:General:PlFjHxuQoH0v7iaVC0QqdxmtQyqY-LQX6D-p",
+  "@type": "general:General",
+  "general:description": {
+    "@type": "general:Description",
+    "gx:name": {
+      "@value": "3D Model in Grafing",
+      "@type": "xsd:string"
+    },
+    "gx:description": {
+      "@value": "town district, with traffic signs",
+      "@type": "xsd:string"
+    }
+  },
+  "general:data": {
+    "@type": "undefined",
```

Comment:
Warum ist der type hier "undefined"?

Reply:
fix in shacl : sh:node general:Data to sh:node general:DataShape